doc/release-notes.md

```diff
@@ -23,492 +26,644 @@ shut down (which might take a few minutes for older versions), then run the
-so called alert keys to notify such nodes in case of severe network issues. This version removes
-the `alert` p2p message and `--alert` option. Internal alerts, partition detection warnings and the
-`--alertnotify` option features remain.
+Block Reward Reallocation
+-------------------------
+This version implements Block Reward Reallocation which was proposed in order
+to slow the growth rate of Dash’s circulating supply by encouraging the
+formation of masternodes and was voted in by the network. The resulting allocation
+will split all non-proposal block rewards 40% toward miners and 60% toward
+masternodes in the end-state once the transition period is complete.
+
+The reallocation will take place gradually over 4.5 years to avoid market volatility
+and transition toward the new end-state allocation with minimal network disruption.
+The proposed changes will occur every three superblock cycles until the reallocation
+is complete, or approximately once per quarter.
+
+Note that this is a hardfork which must be activated by miners. To do this they
+should start creating blocks signalling bit 5 in `version` field of the block header.
+
+Dynamic Activation Thresholds
+-----------------------------
+In Dash we used to use lower thresholds (80% vs 95% in BTC) to activate upgrades
+via BIP9-like mechanism for quite some time. While it's preferable to have as much
+of the network hashrate to signal update readiness as possible this can result in
+quite lengthy upgrades sometimes when one large non-upgraded entity would stale
+the whole progress. Simply lowering thresholds even further can result in network
+upgrades being too fast which can cause some chaos potentially. This version
+implements BIP9-like dynamic activation thresholds which drop from some initial
+level to a minimally acceptable one over time at an increasing rate providing
+a safe non-blocking way of activating proposals.
+
+This mechanism applies to the Block Reward Reallocation proposal mentioned above.
+
+Concentrated Recovery
+---------------------
+In the current system, signature shares are propagated to all LLMQ members,
+until one of them has enough shares collected to recover the signature. Until
+this recovered signature is propagated in the LLMQ, all members will keep
+propagating shares and verifying each one. This causes quite some load on the
+LLMQ, which will be avoided with the new system.
+
+The new system sends all shares to only a single deterministically selected node,
+so that this node can recover the signature and propagate the recovered signature.
+This way only the recovered signature needs to be propagated and verified by all
+members. After sending the share to this single node, each member waits for some
+time and repeats sending it to another deterministically selected member. This
+process is repeated until a recovered signature is finally created and propagated.
+
+The new system is activated with the newly added `SPORK_21_QUORUM_ALL_CONNECTED`.
```

Comment:
I feel like we should justify the addition of a new spork/why this feature is tied to a spork